File: repos_estudiantes_TARQUI_tercera_entrega/ILLANES_VASQUEZ_GABRIEL/Practica 3 Aux/PERSISTENCIA/Ejercicio3/Cliente.java
package Ejercicio3;

public class Cliente {
    private int id;
    private String nombre;
    private int telefono;

    public Cliente(String nombre, int id, int telefono) {
        this.nombre = nombre;
        this.id = id;
        this.telefono = telefono;
    }

    public int getId() {
        return id;
    }

    public String getNombre() {
        return nombre;
    }

    public int getTelefono() {
        return telefono;
    }

    @Override
    public String toString() {
        return "Nombre: " + nombre + ", ID: " + id + ", Telefono: " + telefono;
    }
}

File: repos_estudiantes_ALVAREZ_tercera_entrega/CANAVIRI_ALTAMIRANO_AYLIN_MICHEL/PERSISTENCIADEOBJETOS/EJERCICIO3/Ej3-PYTHON/Cliente.py
class Cliente:
    def __init__(self, id, nombre, telefono):
        self.__id = id
        self.__nombre = nombre
        self.__telefono = telefono

    def get_id(self):
        return self.__id

    def get_nombre(self):
        return self.__nombre

    def get_telefono(self):
        return self.__telefono

    def to_dict(self):
        return {
            "id": self.__id,
            "nombre": self.__nombre,
            "telefono": self.__telefono
        }

    @staticmethod
    def from_dict(data):
        return Cliente(data["id"], data["nombre"], data["telefono"])

    def __str__(self):
        return f"Cliente ID: {self.__id}, Nombre: {self.__nombre}, Teléfono: {self.__telefono}"